atm.js

```diff
@@ -1,21 +1,73 @@
 var checkingBalance = document.querySelector("div#checking_balance");
 checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
 
-var checkingDeposit = function(amount){
+checkingDeposit = function(amount){
   checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+};
+// var checkingBalance = document.querySelector(".checking_balance").innerHTML;
+var extraAmount = 0;
 
-}
+checkingDeposit = function(){
+  inputAmount = parseInt($('.inputField').val());
+  checkingBalance = parseInt( document.querySelector(".checking_balance").innerHTML.replace("$", "") );
+  balanceAmount = inputAmount.checkingBalance;
+  $('.checking_balance').html(balanceAmount);
+};
 
 function checkingWithdrawal(amount){
-
+  var checkingWithdrawal = function(){
+    inputAmount = parseInt($('.inputField').val());
+    if (inputAmount > checkingBalance){
+      checkingBalance = parseInt( document.querySelector(".checking_balance").innerHTML.replace("$", "") );
+      extraAmount = inputAmount - checkingBalance;
+      savingsDeposit2(extraAmount);
+      $('.checking_balance').html('0');
+      return;
+    }
+  };
+  balanceAmount = checkingBalance - inputAmount;
+  $('.checking_balance').html(balanceAmount);
+  checkingBalance = parseInt( document.querySelector(".checking_balance").innerHTML.replace("$", "") );
 }
 
+savingsDeposit2 = function(amount){
+  savingsUpdate();
+  if (amount > savingsBalance){
+    alert('You need mo money');
+  }
+  else if (amount < savingsBalance){
+    sBalanceAmount = savingsBalance - amount;
+    $('.savings_balance').html(sBalanceAmount);
+    savingsUpdate();
+  }
+};
+
 function savingsDeposit(amount){
+  $(".deposit").on('click',checkingDeposit);
```

Comment:
I think having a function with just click events is great and helps make your code more modular! I am a little confused on the savingsUpdate function below since it's nested within this function you would not be able to call it on line 60 below